### PR TITLE
create install and update processes

### PR DESCRIPTION
**Pull request by blockloop**
Install.sh script will be uploaded to Github as a release.
update.sh script will be added to /opt/digitalocean/node_collector/scripts.
/etc/cron.daily/node-collector will be created to execute the update.sh script

### CODE REVIEW

File: packaging/scripts/after_remove.sh [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
+        else
+                echo "Unknown init system" > /dev/stderr
+        fi
+
+        remove_cron
+}
+
+remove_cron() {
+        rm -f "${CRON}"
+	echo "cron removed"
```
Comment (Hoenn) [inline packaging/scripts/after_remove.sh]: `nit` spacing is weird on this line

File: scripts/install.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                *)
+                        not_supported
+                        ;;
+        esac
+}
+
+
+function check_dist() {
+        if [  -f /etc/os-release  ]; then
+                dist=$(awk -F= '$1 == "ID" {gsub("\"", ""); print$2}' /etc/os-release)
```
Comment (Hoenn) [inline scripts/install.sh]: Do you need a space for `print$2` here?

File: packaging/scripts/after_remove.sh (hunk 1 above)
Comment (blockloop) [reply packaging/scripts/after_remove.sh]: ran `retab!` on each of the .sh files and added an .editorconfig file to fix this issue. Pushed a separate commit for it.

File: scripts/install.sh (hunk 2 above)
Comment (blockloop) [reply scripts/install.sh]: No. I also stole this func from [here](https://github.internal.digitalocean.com/digitalocean/agent-packages/blob/b0c86652c2138f7e38f476901a197b9ca91b5ef5/agent_do_com/install.sh#L19)

**Pull request merged**